packages/design-system/src/scalars/components/radio-group/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useId } from "react";
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group";
+import { FormLabel } from "@/scalars/components/form-label";
+import { cn } from "@/scalars/lib/utils";
+
+export interface RadioGroupProps
+  extends React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Root> {
+  "aria-label"?: string;
```

Comment:
if ` RadioGroupPrimitive.Root` doesn't include this, we should inherit from the HTML input element and make this out the box. There's this param, but could be dozen more like event handlers, etc and we should not handle each one of them via props this way unless there's no other way to do it

Reply:
Removed ( RadioGroupPrimitive.Root has aria-label support)